### PR TITLE
Adding app metadata

### PR DESCRIPTION
**Pull request by tomahawk28**
For testing https://github.com/giantswarm/giantswarm/issues/13973

## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/chart-operator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 apiVersion: v1
 name: chart-operator
-description: A Helm chart for the chart-operator
-home: https://github.com/giantswarm/chart-operator
-appVersion: [[ .AppVersion ]]
 version: [[ .Version ]]
+appVersion: [[ .AppVersion ]]
+home: https://github.com/giantswarm/chart-operator
```
Comment (rossf7) [inline helm/chart-operator/Chart.yaml:5]: What about adding icon with the GS logo as well?

**Review by rossf7 — COMMENTED** [review]
Looks good. Just wondering if we can add our GS icon. 

@cokiengchiara @oponder Do you have a link for that?

File: helm/chart-operator/Chart.yaml (hunk 1 above)
Comment (oponder) [reply helm/chart-operator/Chart.yaml:5]: not super important for these control plane apps, also we'll have a cool auto generated icon based on the first letters of the app name coming up soon for apps that don't define an icon.

unless you wanna make something clever or cute that builds on the giant swarm logo, for identifying it and building an identity for chart operator... for now I would leave it without an icon.

Comment (rossf7) [reply helm/chart-operator/Chart.yaml:5]: Thanks @oponder cool lets leave it then.

**Comment by tomahawk28** [comment]
going with a ping

**Pull request merged**